NAMESPACE

```diff
@@ -68,7 +69,11 @@ exportMethods(toGRanges)
 import(GenomicRanges)
 import(IRanges)
 import(methods)
+import(Biostrings)
 importClassesFrom(graph,graphNEL)
+importFrom(Biostrings,oligonucleotideFrequency)
+importFrom(stats,binom.test)
+importFrom(universalmotif,shuffle_sequences)
```

Comment:
You are importing a new package here. Too much dependencies already make the package hard to maintain. Please consider to implement the function by your own code.